src/stories/Library/material-search/Errors/MaterialSearchBaseError.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { ReactComponent as WarningIcon } from "../../../../public/icons/basic/icon-warning.svg";
+
+interface MaterialSearchBaseErrorProps {
+  children: React.ReactNode;
+}
+
+const MaterialSearchBaseError: React.FC<MaterialSearchBaseErrorProps> = ({
+  children,
+}) => {
+  return (
+    <div className="material-search__error">
+      <div className="material-search__error-header">
+        <WarningIcon className="material-search__error-icon" />
+        <div>This material is outdated.</div>
```

Comment:
Consider marking this with an actual heading level to signal the priority over the following content/description.